validator/sawtooth_validator/journal/chain.py

```diff
@@ -505,6 +505,37 @@ def _verify_block(self, blkw):
         self._blocks_processing[blkw.block.header_signature] = validator
         self._executor.submit(validator.run)
 
+    def _remove_block_and_verify_descendants(self, parent_block):
```

Comment:
If it wasn't for link, I could name it `_remove_block_from_pending_queue_and_process_descendants()` ;)